app/views/news/index.html.haml

```diff
@@ -1,9 +1,34 @@
 #news_display
-  %h1.blog-index-head
-    = t(".bike_index_blog")
-    = link_to image_tag('feed_icon.svg'), news_index_url(format: 'atom'), { title: t('.bike_index_blog_feed' )}
+  .row
+    .col-md-12.col-lg-8.col-xl-9
+      %h1
+        = t(".bike_index_blog")
+        = link_to image_tag('feed_icon.svg', class: "rss-svg"),
+          news_index_url(format: 'atom'),
+          title: t('.bike_index_blog_feed'),
+          class: "blog-rss-link"
 
+    .col-md-12.col-lg-3.blog-language-selection-container
+      View posts in
+      = form_tag nil, method: :get,
+        class: "blog-language-selection", id: "js-blog-language-selection" do
+        = select_tag :language,
+          options_for_select(blog_languages, selected: params[:language].presence),
+          onchange: "this.form.submit()"
   %hr
+
+:javascript
+  const setBlogLanguageSelection = () => {
```

Comment:
I'd also like the parameters to use "en"/"nl" rather than 0 and 1.

I don't think the language sticking when using the back button is a large enough concern to add this JS (concerns I have with this JS are that it will break older browsers, and that using "en"/"nl" will make it more complicated).

Reply:
It looks pretty rough without it. Happy to convert to ES5 if that'd be better...

Reply:
Why does it look rough? If it's just the back browser state thing, I don't think that's much of a concern. When I navigate backward, I expect the state of select & text boxes to be sometimes funky.

Reply:
> the state of select & text boxes to be sometimes funky.

That's what I'm describing as rough